src/lib/components/BeforeAfter.react.js

```diff
@@ -1,55 +1,153 @@
+                direction={direction}
+                keyboard={keyboard}
+            >
+                <img
+                    slot="first"
+                    width={width}
+                    height={height}
+                    src={before}
+                    style={before_style}
+                    className={beforeClassName}
+                />
+                <img
+                    slot="second"
+                    width={width}
+                    height={height}
+                    src={after}
+                    style={after_style}
+                    className={afterClassName}
+                />
+            </ImgComparisonSlider>
+        </div>
+    );
+};
+
+
+BeforeAfter.defaultProps = {
+    width: '100%',
+    height: 'auto',
+    hover: true,
+    value: 50,
+    direction: 'horizontal',
+    keyboard: 'enabled',
+};
 
-    render() {
-        return (
-            <BeforeAfterSlider {...this.props}/>
-        );
-    }
+BeforeAfter.propTypes = {
+    /**
+     * The ID used to identify this component in Dash callbacks.
+     */
+    id: PropTypes.string,
 
-}
+    /**
+     * Before image src
+     */
+    before: PropTypes.string.isRequired,
 
-BeforeAfter.propTypes = {
+    /**
+     * After image src
+     */
+    after: PropTypes.string.isRequired,
 
-    before: PropTypes.string,
+    /**
+     *  image height - default "auto" for responsive images.
+     */
+    height: PropTypes.string,
 
-    after: PropTypes.string,
+    /**
+     * image width - default "100%" for responsive images.
+     */
+    width: PropTypes.string,
 
-    width: PropTypes.number,
+    /**
+     * Automatic slide on mouse over.
+     */
+    hover: PropTypes.bool,
 
-    height: PropTypes.number,
+    /**
+     * The divider position can be specified as a percentage. 0 to 100
+     */
+    value: PropTypes.number,
 
-    defaultProgress: PropTypes.number,
+    /**
+     * Set slider direction
+     */
+    direction: PropTypes.oneOf(['horizontal', 'vertical']),
 
-    beforeClassName: PropTypes.string,
+    /**
+     * Enable/disable slider position control with the keyboard
+     */
+    keyboard: PropTypes.oneOf(['enabled', 'disabled']),
 
-    afterClassName: PropTypes.string,
+    /**
+     * Defines CSS styles of the before image.
+     */
+    before_style: PropTypes.object,
 
-    beforeProps: PropTypes.object,
+    /**
+     * Defines CSS styles of the after image.
+     */
+    after_style: PropTypes.object,
 
-    afterProps: PropTypes.object,
+    /**
+     * className of before image. Often used with CSS to style elements with common properties.
+     */
+    beforeClassName: PropTypes.string,
 
-    // Dash props.
+    /**
+     * className of after image. Often used with CSS to style elements with common properties.
+     */
+    afterClassName: PropTypes.string,
 
     /**
-     * The ID used to identify this component in Dash callbacks
+     * **DEPRECATED** Use `value` instead.
```

Comment:
Now that we are moving to a new component, I think we should just remove the property. As we discussed the changes (e.g. switching before/after order) are breaking already, so I'll just include that in the release note.